include/deal.II/cgal/surface_mesh.h

```diff
@@ -25,12 +25,24 @@
 #include <deal.II/cgal/utilities.h>
 
 #ifdef DEAL_II_WITH_CGAL
+#  include <CGAL/Polygon_mesh_processing/corefinement.h>
+#  include <CGAL/Polygon_mesh_processing/triangulate_faces.h>
 #  include <CGAL/Surface_mesh.h>
+#  include <CGAL/Triangulation_3.h>
+#  include <CGAL/convex_hull_3.h>
+
 
 DEAL_II_NAMESPACE_OPEN
 
 namespace CGALWrappers
 {
```

Comment:
Add a bit of documentation, with some examples from the CGAL web site, or computed with the tests.